src/client.cpp

```diff
@@ -821,9 +823,12 @@ void Client::initLocalMapSaving(const Address &address,
 
 	fs::CreateAllDirs(world_path);
 
-	m_localdb = new Database_SQLite3(world_path);
+	Settings conf;
+	conf.set("sqlite_journal_mode", g_settings->get("sqlite_journal_mode"));
+	m_localdb = new Database_SQLite3(world_path, conf);
```

Comment:
I assume users (who are mostly not server operators) will prefer the same setting for all locally-saved maps; a setting that can be configured in a single place. If it is configurable per local-save-database, then I think hardly anybody, if anybody at all, will ever use that feature. Also, it means that every local-save-database will need its own `world.mt` to store the current settings, and if ever a user wants to use a different mode, he'll have to check and adapt every `world.mt` of every server he uses.